infrastructure/fargate_cluster.yml

```diff
@@ -0,0 +1,368 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Is this taken from a standard template documented somewhere? That's a lot of networking that I do not understand :D